lib/urlwatch/jobs.py

```diff
@@ -209,7 +209,7 @@ class UrlJob(Job):
     __required__ = ('url',)
     __optional__ = ('cookies', 'data', 'method', 'ssl_no_verify', 'ignore_cached', 'http_proxy', 'https_proxy',
                     'headers', 'ignore_connection_errors', 'ignore_http_error_codes', 'encoding', 'timeout',
-                    'ignore_timeout_errors', 'ignore_too_many_redirects')
+                    'ignore_timeout_errors', 'ignore_too_many_redirects', 'content_type', 'mime_type')
```

Comment:
What's the difference between `content_type` and `mime_type` here?

Reply:
- `content_type` is the raw response header including optional encoding information.
 - `mime_type` is the first part before optional ";" and doesn't contain optional encoding information, which is already taken into account when decoding the response. this is what you'd usually use.

Reply:
These are settings from the jobs list YAML, you don't need to specify them here? (users are not supposed to "set" those values in the job configuration, right?)

Reply:
Ok, nevermind, I understood how you plan to use this now... Basically that's a kind of "separate" kind of key that we (I think) haven't had yet where we want the job to obtain those properties (so it can be used for the filter matching) but it's not something we want the user to specify via the config variable (we'd otherwise overwrite it?).

Reply:
Right, I included them here so the automatic matching takes these two dictionary keys in `MATCH` into account.

After your remarks about defaults in `ShellJob`s maybe it should be possible to configure it (one can create say screenshots with a script).